### PR TITLE
feat: handle default remote drives

### PR DESCRIPTION
**Pull request by gpuente**
## Description:

-  handle default remote drives

### CODE REVIEW

File: src/server/error.ts [hunk 1]
```diff
@@ -33,3 +33,9 @@ export class MissingOperationError extends OperationError {
         super('MISSING', operation, `Missing operation on index ${index}`);
     }
 }
+
+export class DriveAlreadyExistsError extends Error {
+    constructor() {
```
Comment (acaldas) [inline src/server/error.ts:38]: Let's also store the drive id on the error.

**Review by acaldas — COMMENTED** [review]
Great!
Can you add an option to only allow the default drives? On startup, if `getDrives` returns drives that are not part of the default drives they would be removed.
This is what we want for the envs like Arbgrants and Maker

File: src/server/error.ts (hunk 1 above)
Comment (gpuente) [reply src/server/error.ts]: sure! added here: [e1097a7](https://github.com/powerhouse-inc/document-drive/pull/277/commits/e1097a75a7b8c4475310127217252cc2a3e8cae5)

**Comment by gpuente** [comment]
> Great! Can you add an option to only allow the default drives? On startup, if `getDrives` returns drives that are not part of the default drives they would be removed. This is what we want for the envs like Arbgrants and Maker

there's another ticket for this one, so I'm working on it in a different PR 👍🏻

**Review by acaldas — APPROVED** [review]

**Pull request merged**